pytroll_collectors/tests/test_segments.py

```diff
@@ -1319,22 +1329,23 @@ def test_floor_time_2_minutes_with_seconds(self):
                                                                            start_time.day, start_time.hour,
                                                                            start_time.minute, 42)
             self.himawari_ini_message.adjust_time_by_flooring()
-            self.assertEqual(self.himawari_ini_message.metadata['start_time'].minute, 28)
-            self.assertEqual(self.himawari_ini_message.metadata['start_time'].second, 0)
+            assert self.himawari_ini_message.metadata['start_time'].minute == 28
+            assert self.himawari_ini_message.metadata['start_time'].second == 0
 
     def test_floor_time_without_group_by_minutes_does_not_change_time(self):
         """Test that flooring the time without group_by_minutes defined keeps the time intact."""
         with patch.dict(CONFIG_INI_HIMAWARI, {'group_by_minutes': None}):
             self.himawari_ini = SegmentGatherer(CONFIG_INI_HIMAWARI)
             self.himawari_ini_message = self.himawari_ini.message_from_posttroll(self.himawari_msg)
             self.himawari_ini_message.adjust_time_by_flooring()
-            self.assertEqual(self.himawari_ini_message.metadata['start_time'].minute, 29)
+            assert self.himawari_ini_message.metadata['start_time'].minute == 29
 
 
-class TestFlooringMultiplePatterns(unittest.TestCase):
+class TestFlooringMultiplePatterns:
     """Test flooring."""
 
-    def setUp(self):
+    @pytest.fixture(autouse=True)
+    def setup(self):
```

Comment:
using `setup_method` should allow getting rid of the autouse call

Reply:
Fixed in https://github.com/pytroll/pytroll-collectors/pull/133/commits/15ea90e4cf7c333c10f204bd1d3954ee736e1fca